tests/graphql_spec.rs

```diff
@@ -23,16 +23,54 @@ mod graphql_mock;
 
 static INIT: Once = Once::new();
 
+#[derive(Debug, Clone)]
+struct Source {
+  sdl: String,
+  tag: String,
```

Comment:
wouldn't it be better if use enum with explicit variants instead of arbitrary string?

Reply:
Agree with @meskill